src/components/CheckboxInput/CheckboxInput.module.css

```diff
@@ -42,7 +42,7 @@
 }
 
 /**
- * The visible checkbox svg icon element.
+ * 1) The visible checkbox svg icon element.
```

Comment:
We don't need the number here.
```suggestion
 * The visible checkbox svg icon element.
```

Reply:
removed thanks!